pkg/utils/files.go

```diff
@@ -0,0 +1,37 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+func FindFiles(humanOutput bool, args ...string) ([]string, error) {
+	var files []string
+	for _, fileOrDir := range args {
+		if info, err := os.Stat(fileOrDir); err != nil {
+			return nil, err
+		} else if info.IsDir() {
+			if entries, err := os.ReadDir(fileOrDir); err != nil {
+				return nil, err
+			} else {
+				for _, entry := range entries {
+					if sub, err := FindFiles(humanOutput, filepath.Join(fileOrDir, entry.Name())); err != nil {
+						return nil, err
+					} else {
+						files = append(files, sub...)
+					}
+				}
+			}
+		} else {
+			ext := strings.ToLower(filepath.Ext(fileOrDir))
+			if ext == ".json" || ext == ".yaml" || ext == ".yml" {
+				files = append(files, fileOrDir)
+			} else if humanOutput {
+				fmt.Printf("skipping %v since it is not json or yaml\n", fileOrDir)
+			}
+		}
+	}
+	return files, nil
+}
```

Comment:
There must be a way to write a test for that, can you add one? Thanks!

Reply:
Will do

Reply:
@apelisse added tests for `FindFiles` how does this look ?